core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -224,7 +230,16 @@ private boolean startSyncInternal(AlluxioURI syncPoint) throws InvalidPathExcept
           throw new UnsupportedOperationException("Active Syncing is not supported on this UFS type"
               + ufsResource.get().getUnderFSType());
         }
-        ufsResource.get().startSync(resolution.getUri());
+        Future<?> syncFuture = mExecutorService.submit(
+            () -> {
+              try {
+                ufsResource.get().startSync(resolution.getUri());
```

Comment:
also update message in StartSyncCommand.java with how to check the status of sync instead of "This may take a while..." :)